code/modules/reagents/chemistry/reagents/impure_reagents/impure_toxin_reagents.dm

```diff
@@ -0,0 +1,68 @@
+//Reagents produced by metabolising/reacting fermichems inoptimally these specifically are for toxins
+//Inverse = Splitting
+//Invert = Whole conversion
+//Failed = End reaction below purity_min
+
+////////////////////TOXINS///////////////////////////
+
+//Lipolicide
+//impure
+/datum/reagent/impurity/ipecacide
+	name = "Ipecacide"
+	description = "An extremely gross substance that induces vomiting. It is produced when Lipolicide reactions are impure."
+	ph = 7
+	liver_damage = 0
+
+/datum/reagent/impurity/ipecacide/on_mob_metabolize(mob/living/carbon/owner)
+	owner.adjust_disgust(100)
+	return ..()
+
+
+//Formaldehyde
+//impure
+/datum/reagent/impurity/methanol
+	name = "Methanol"//Chemically related to formaldehyde, do not drink this
+	description = "A light, colourless liquid with a distinct smell. Ingestion can lead to blindness. It is a byproduct of organisms processing impure Formaldehyde."
+	reagent_state = LIQUID
+	color = "#aae7e4"
+	ph = 7
+
+/datum/reagent/impurity/methanol/on_mob_life(mob/living/carbon/owner)
+	var/obj/item/organ/eyes/eyes = owner.getorganslot(ORGAN_SLOT_EYES)
+	eyes.applyOrganDamage(0.5)
+	return ..()
+
+
+//Chloral Hydrate
+//impure
+/datum/reagent/impurity/chloral
+	name = "Chloral"
+	description = "An oily, colorless and slightly toxic liquid. It is produced when impure chloral hydrate is broken down inside an organism."
+	reagent_state = LIQUID
+	color = "#387774"
+	ph = 7
+	liver_damage = 0
+
+/datum/reagent/impurity/chloral/on_mob_life(mob/living/carbon/owner, delta_time)
+	owner.adjustToxLoss(1 * REM * delta_time, 0)
+	return ..()
+
+
+//Mindbreaker Toxin
+//impure
+/datum/reagent/impurity/rosenol
+	name = "Rosenol"
+	description = "A strange, blue liquid that is produced during impure mindbreaker toxin reactions. Historically it has been abused to write poetry."
+	reagent_state = LIQUID
+	color = "#0963ad"
+	ph = 7
+	liver_damage = 0
+
+/datum/reagent/impurity/rosenol/on_mob_life(mob/living/carbon/owner, delta_time)
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	if(DT_PROB(4.0, delta_time))
+		owner.manual_emote("clicks with [owner.p_their()] tongue.")
+		owner.say("Noice.")
+	if(DT_PROB(2.0, delta_time))
+		owner.say(pick("Ah! That was a mistake!", "Horrible.", "Watch out everybody, the potato is really hot.", "When I was six I ate a bag of plums.", "And if there is one thing I can't stand its tomatoes.", "And if there is one thing I love its tomatoes.", "We had a captain who was so strict, you weren't allowed to breathe in their presence.", "The unrobust ones just used to keel over and die, you'd hear them going down behind you."), forced = /datum/reagent/impurity/rosenol)
```

Comment:
```suggestion
		owner.say(pick("Ah! That was a mistake!", "Horrible.", "Watch out everybody, the potato is really hot.", "When I was six I ate a bag of plums.", "And if there is one thing I can't stand, it's tomatoes.", "And if there is one thing I love, it's tomatoes.", "We had a captain who was so strict, you weren't allowed to breathe in their presence.", "The unrobust ones just used to keel over and die, you'd hear them going down behind you."), forced = /datum/reagent/impurity/rosenol)
```
I'm not sure if there should or shouldn't be a comma after "watch out", so I've left it as it was.

Reply:
> 
> 
> Also - if it's not too much trouble, could I ask you to run all of the changed recipes in the debug machine using standard conditions and paste the results here?

No problem, I divided into 3 categories here:
- **Cold Reactions** (Standard debug machine settings)
- **Hot Reactions** (Debug machine temperature forced to 450°K - 900°K)
- **Special** (Different mechanics, lower amounts)


### Cold Reactions
"Reacting /datum/chemical_reaction/medicine/seiver starting pH: 6.48333 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/medicine/seiver final temperature = 0, ph = 1.7, time taken = 2s."
"Reaction has a product /datum/reagent/medicine/c2/**seiver** 60u purity of 0.710084"

"Reacting /datum/chemical_reaction/itching_powder starting pH: 8.40729 index 2 of 14"
"Reaction completed for /datum/chemical_reaction/itching_powder final temperature = 279.964, ph = 9.65345, time taken = 2s."
"Reaction has a product /datum/reagent/toxin/**itching_powder** 31.59u purity of 0.889431"

"Reacting /datum/chemical_reaction/sulfonal starting pH: 7.83861 index 3 of 14"
"Reaction completed for /datum/chemical_reaction/sulfonal final temperature = 635.601, ph = 11, time taken = 4s."
"Reaction has a product /datum/reagent/toxin/**sulfonal** 60u purity of 0.806375"

"Reacting /datum/chemical_reaction/lipolicide starting pH: 10.0151 index 4 of 14"
"Reaction completed for /datum/chemical_reaction/lipolicide final temperature = 902.38, ph = 8.5, time taken = 4s."
"Reaction has a product /datum/reagent/toxin/**lipolicide** 58.8u purity of 0.799186"

"Reacting /datum/chemical_reaction/mutagen starting pH: 7.66029 index 5 of 14"
"Reaction completed for /datum/chemical_reaction/mutagen final temperature = 780.952, ph = 2.4, time taken = 4s."
"Reaction has a product /datum/reagent/toxin/**mutagen** 60u purity of 0.943691"

"Reacting /datum/chemical_reaction/lexorin starting pH: 1.92572 index 6 of 14"
"Reaction completed for /datum/chemical_reaction/lexorin final temperature = 160.237, ph = 1.3, time taken = 10s."
"Reaction has a product /datum/reagent/toxin/**lexorin** 60u purity of 0.763776"

"Reacting /datum/chemical_reaction/mutetoxin starting pH: 5.2504 index 7 of 14"
"Reaction completed for /datum/chemical_reaction/mutetoxin final temperature = 282.995, ph = 12, time taken = 6s."
"Reaction has a product /datum/reagent/toxin/**mutetoxin** 40u purity of 0.8526"

"Reacting /datum/chemical_reaction/mindbreaker starting pH: 6.65321 index 8 of 14"
"Reaction completed for /datum/chemical_reaction/mindbreaker final temperature = 630.674, ph = 5.2396, time taken = 6s."
"Reaction has a product /datum/reagent/toxin/**mindbreaker** 88.02u purity of 0.880182"

"Reacting /datum/chemical_reaction/heparin starting pH: 8.17871 index 9 of 14"
"Reaction completed for /datum/chemical_reaction/heparin final temperature = 813.667, ph = 10.9996, time taken = 8s."
"Reaction has a product /datum/reagent/toxin/**heparin** 75.34u purity of 0.771508"

"Reacting /datum/chemical_reaction/anacea starting pH: 7.89096 index 10 of 14"
"Reaction completed for /datum/chemical_reaction/anacea final temperature = 682.982, ph = 11, time taken = 4s."
"Reaction has a product /datum/reagent/toxin/**anacea** 60u purity of 0.808513"

"Reacting /datum/chemical_reaction/mimesbane starting pH: 8.77827 index 11 of 14"
"Reaction completed for /datum/chemical_reaction/mimesbane final temperature = 291.886, ph = 0, time taken = 106s."
"Reaction has a product /datum/reagent/toxin/**mimesbane** 60u purity of 0.763624"

"Reacting /datum/chemical_reaction/bonehurtingjuice starting pH: 5.61287 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/**bonehurtingjuice** final temperature = 99.8757, ph = 2.73955, time taken = 10s."
"Reaction has a product /datum/reagent/toxin/bonehurtingjuice 98.87u purity of 0.745497"

### Hot Reactions
"Reacting /datum/chemical_reaction/cyanide starting pH: 8.74728 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/cyanide final temperature = 450, ph = 13.0611, time taken = 6s."
"Reaction has a product /datum/reagent/toxin/**cyanide** 59u purity of 0.723989"

"Reacting /datum/chemical_reaction/formaldehyde starting pH: 7.68462 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/formaldehyde final temperature = 460, ph = 0, time taken = 18s."
"Reaction has a product /datum/reagent/toxin/**formaldehyde** 59u purity of 0.931867"

"Reacting /datum/chemical_reaction/fentanyl starting pH: 9.30748 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/fentanyl final temperature = 700, ph = 12.3417, time taken = 8s."
"Reaction has a product /datum/reagent/toxin/**fentanyl** 19u purity of 0.696229"

"Reacting /datum/chemical_reaction/facid starting pH: 1.72817 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/facid final temperature = 900, ph = 0, time taken = 2s."
"Reaction has a product /datum/reagent/toxin/acid/**fluacid** 68.48u purity of 0.697389"

"Reacting /datum/chemical_reaction/nitracid starting pH: 2.67567 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/nitracid final temperature = 900, ph = 0, time taken = 0s."
"Reaction has a product /datum/reagent/toxin/acid/**nitracid** 39.33u purity of 0.725699"

"Reacting /datum/chemical_reaction/rotatium starting pH: 8.85961 index 1 of 1"
"Reaction completed for /datum/chemical_reaction/rotatium final temperature = 450, ph = 10.2, time taken = 4s."
"Reaction has a product /datum/reagent/toxin/**rotatium** 30u purity of 0.866555"

### Special
These three are always produced in very small batches (because of the low yield)
In context of that, the debugger is at 300°K and 10u per reagent.

"Reacting /datum/chemical_reaction/chloralhydrate starting pH: 8.9657 index 1 of 3"
"Reaction completed for /datum/chemical_reaction/chloralhydrate final temperature = 567.674, ph = 13, time taken = 0s."
"Reaction has a product /datum/reagent/toxin/**chloralhydrate** 5u purity of 0.82"

"Reacting /datum/chemical_reaction/zombiepowder starting pH: 7.32838 index 2 of 3"
"Reaction completed for /datum/chemical_reaction/zombiepowder final temperature = 519.112, ph = 10.5, time taken = 0s."
"Reaction has a product /datum/reagent/toxin/**zombiepowder** 10u purity of 1"

"Reacting /datum/chemical_reaction/ghoulpowder starting pH: 12.2465 index 3 of 3"
"Reaction completed for /datum/chemical_reaction/ghoulpowder final temperature = 558.018, ph = 13.82, time taken = 0s."
"Reaction has a product /datum/reagent/toxin/**ghoulpowder** 10u purity of 1"


Zombie powder and ghoul powder are so hard to get in good amounts, I didn't want the chemist to lose even more